### DISCUSSION TITLE
Omit wrappers of areabricks

### DISCUSSION THREAD

**Discussion by zangab** [pimcore/pimcore#10898] [Q&A / Support] [answered]
Hi,
all areabricks are beeing rendered with an additional div wrapper on the fronend.
I wonder why this is necessary? and how can i get rid of them? or pass any css-classes to that??
Had no success with the `manual` mode.

The `<main>` includes the `pimcore_areablock(...)` part.
The actual `view.html.twig` of that `NewsCarousel` contains the `<section>` only.

Thx for info.

<img width="532" alt="pimcore-areabricks" src="https://user-images.githubusercontent.com/9076592/143312670-906cb177-dd43-45f8-8199-b9900e9ff826.png">
?

**Comment by dvesh3** [comment, marked as answer]
Hi @zangab, you can return empty string from `getHtmlTagOpen()` & `getHtmlTagClose()` in your brick class to avoid wrappers https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Areablock/Bricks.html#page_Methods-on-the-brick-class

**Reply by zangab** [reply]
Aaaaaah, didn't see that one, nice, thx. 👍
